packages/client/components/ActivityLibrary/ActivityLibrary.tsx

```diff
@@ -74,8 +142,25 @@ export const ActivityLibrary = (props: Props) => {
       <ActivityLibrarySideBar />
       <div>
         <SearchBar searchQuery={searchQuery} onChange={onQueryChange} />
+        <div className='ml-2 flex gap-x-2'>
+          {(Object.keys(CATEGORY_ID_TO_NAME) as Array<CategoryID>).map((category) => (
+            <Link
```

Comment:
+1 there's a `NavLink` that can be styled depending if the link is active or not automatically, more [here](https://v5.reactrouter.com/web/api/NavLink). just a fyi, not a merge blocker by any means

Reply:
TIL! Unfortunately there seems to be an issue with our version of @types/react-router-dom that prevents us from using the `isActive` prop, and I don't want that the necessary upgrade to creep into this PR.